packages/html-reporter/src/metadataView.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+  Copyright (c) Microsoft Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import * as React from 'react';
+import './colors.css';
+import './common.css';
+import * as icons from './icons';
+import { Metadata } from './index';
+import { AutoChip } from './chip';
+import './reportView.css';
+import './theme.css';
+
+export const MetadataView: React.FC<Metadata> = metadata => {
+  return (
+    <AutoChip header={
+      <span>
+        {metadata['revision.id'] && <span style={{ float: 'right', fontFamily: 'var(--monospace-font)' }}>
+          {metadata['revision.id'].slice(0, 7)}
+        </span>}
+        {metadata['revision.subject'] && metadata['revision.subject'] || 'no subject>'}
+        {!metadata['revision.subject'] && 'Commit metainfo'}
+      </span>} initialExpanded={false}>
+      {metadata['revision.subject'] &&
+        <MetadatViewItem
+          testId='revision.subject'
+          content={<span>{metadata['revision.subject']}</span>}
+        />
+      }
+      {metadata['revision.id'] &&
+        <MetadatViewItem
+          testId='revision.id'
+          content={<span style={{ fontFamily: 'var(--monospace-font)' }}>{metadata['revision.id']}</span>}
+          href={metadata['revision.link']}
+          icon='commit'
+        />
+      }
+      {(metadata['revision.author'] || metadata['revision.email']) &&
+        <MetadatViewItem
+          content={(
+            metadata['revision.author'] && metadata['revision.email']
+              ? <>{metadata['revision.author']} {metadata['revision.email']}</>
+              : (metadata['revision.author'] || metadata['revision.email'])
+            )!}
```

Comment:
```suggestion
          content={`${metadata['revision.author'] ${metadata['revision.email']}`}
```

There's no more <br/> being conditionally inserted, so this can be simplified.

Reply:
Following up with you offline!